src/cli/index.ts

```diff
@@ -36,8 +36,17 @@ program.hook('preAction', (thisCommand) => {
 program
   .command('init [path]')
   .description('Initialize OpenSpec in your project')
-  .action(async (targetPath = '.') => {
+  .option('--tools <tools>', 'Comma-separated list of AI tools to configure (claude,cursor,opencode,kilocode,windsurf)')
```

Comment:
Is it possible to use `AI_TOOLS` in the description or so? Just trying to think about how AI tools will figure out what tools are currently supported. A static list here is probably not great.